eq-author/src/components/AnswerPicker/index.js

```diff
@@ -139,15 +137,17 @@ const QuestionPicker = ({
                   {validTypes.map((type) => (
                     <Type key={type}>{type}</Type>
                   ))}
-                </Types>) : ('')
-              }
-              {
-                showSearch ? (
-                  //TODO needs onChange prop setup and passed here
-                  // see example in App/QuestionnairesPage/QuestionnairesView/Header
-                  <SearchBar size="large"/>
-                ) : ('')
-              }
+                </Types>
+              ) : (
+                ""
+              )}
+              {showSearch ? (
+                //TODO needs onChange prop setup and passed here
+                // see example in App/QuestionnairesPage/QuestionnairesView/Header
```

Comment:
```suggestion
                //TODO needs onChange prop setup and passed here
                // see example in App/QuestionnairesPage/QuestionnairesView/Header
```

Is this comment still needed?

Reply:
Probably not no, I don't think we need the search bar here either. I'll take those out this morning; good shout Tom.